playbook/app/pb_kits/playbook/pb_table/docs/_table_striped.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= pb_rails("table", props: { striped: true }) do %>
```

Comment:
I would add size:"sm" on this rails doc example so rails and react doc examples look the same. Looks like default for the size prop is different between react and rails, but that's a separate issue. For this usecase just adding size:"sm" should be enough!

Reply:
Thank you. Will do it!